src/components/Image/Image.st.css

```diff
@@ -12,11 +12,19 @@
 }
 
 .root {
-  -st-states: preload, loaded;
+  -st-states: resize(enum(contain, cover)), preload, loaded;
```

Comment:
I think it might be better to use classes instead of stylable states.
In general, in our library.

Reply:
I saw a lot of usages (about 30+- components) in our library, maybe worth mentioning it in a guideline doc?

Reply:
yeah, we used states up until now.
It's something that we're just starting to realize and understand